weather/templates/weather/partials/alert-link.html

```diff
@@ -0,0 +1,27 @@
+{% comment %}
+# Expected inputs:
+# - alertID        - the ID used to link to this specific alert, or null to link
+# to the alerts tab
+# - alertType      - the event type, such as "Severe Thunderstorm Warning"
+# - alertLevel - the severity level of the event type, such as "Warning"
+{% endcomment %}
```

Comment:
I know that the python type system is rather new, so I'm willing to be shot down on this idea, but how do you feel about us using types rather than comments?

Reply:
I've been out of the game for long enough that I didn't know this was an option! Let's discuss online